android/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for Android enables you to integrate your SPA with our services.
+There are two methods of UI integration:
+- Embedded - With this option, each app has to have its own login User Interface (UI).
+  When a user attempts to log in to your application or site, they authenticate directly within the currently used application instead of being directed to a centralized web application to authenticate the user.
+- OIDC Login - Often referred to as centralized login,
```

Comment:
Might be more consumable as a few bullet points, listing the options for the centralized UI. 
It's a bit of a super-sentence otherwise.

Reply:
Done.